lib/MusicBrainz/Server/Edit/Medium/Edit.pm

```diff
@@ -192,6 +192,16 @@ sub initialize
                 $data->{old}{tracklist} = $old;
                 $data->{new}{tracklist} = $new;
             }
+        } else {
+            $self->c->model('Track')->load_for_mediums($entity);
+            $self->c->model('ArtistCredit')->load($entity->all_tracks);
+
+            my $old = tracks_to_hash($entity->tracks);
+            my $new = [];
+
+            $data->{old}{tracklist} = $old;
+            $data->{new}{tracklist} = $new;
+
```

Comment:
```suggestion
```

You could probably also assign the `$old` / `$new` values directly (unless you feel the variables make it clearer)